appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -103,6 +103,19 @@
 		}
 	}
 
+	private void TeasingSonarQube()
+	{
+		try
+		{
+			final String password = "Secret";
+			System.out.print("The password = " + password);
+		}
+		catch (Throwable e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS1181)